sklearn/metrics/_classification.py

```diff
@@ -521,6 +521,119 @@ def multilabel_confusion_matrix(y_true, y_pred, sample_weight=None,
     return np.array([tn, fp, fn, tp]).T.reshape(-1, 2, 2)
 
 
+def most_confused_classes(y_true, y_pred, labels=None,
+                          max_rows=None, normalize=None):
+    """Most confused classes.
+
+    In multiclass and binary classification, this function computes the classes
+    which have been confused the most often by the classifier.
+
+    Parameters
+    ----------
+
+    y_true: array, shape = [n_samples]
+        Ground truth (correct) target values.
+
+    y_pred : array, shape = [n_samples]
+        Estimated targets as returned by a classifier.
+
+    labels : array, shape = [n_classes], optional
+        List of labels to index the confusion matrix. This may be used to
+        reorder or select a subset of labels.
+        If none is given, those that appear at least once in ``y_true`` or
+        ``y_pred`` are used in sorted order.
+
+    max_rows : int, optional
+        Maximum number of pairs of classes to return. Corresponds to the
+        number of rows in most_confused.
+        If none is given, all rows are returned.
+
+    normalize : {'true', 'pred', 'all'}, default=None
+        Normalizes most confused classes over the true, predicted conditions
+        or all the population. If None, most confused classes will not be
+        normalized.
+        For normalize='all', the sum of the counts of the most_confused matrix
+        is equal to 1 - accuracy_score.
+
+    Returns
+    -------
+    most_confused : list (of length n_pairs) of tuples (of length 3)
```

Comment:
`n_pairs` is more or less the same as `max_rows`. Why the change in nomenclature?